diff --git a/node_modules/bumpp/dist/chunk-ZY4OVS4D.mjs b/node_modules/bumpp/dist/chunk-YUHUYJBY.mjs
similarity index 99%
rename from node_modules/bumpp/dist/chunk-ZY4OVS4D.mjs
rename to node_modules/bumpp/dist/chunk-YUHUYJBY.mjs
index 1857e2c..17a5614 100644
--- a/node_modules/bumpp/dist/chunk-ZY4OVS4D.mjs
+++ b/node_modules/bumpp/dist/chunk-YUHUYJBY.mjs
@@ -914,6 +914,10 @@ function writeTextFile(file) {
 function isManifest(obj) {
   return obj && typeof obj === "object" && isOptionalString(obj.name) && isOptionalString(obj.version) && isOptionalString(obj.description);
 }
+function isPackageLockManifest(manifest) {
+  var _a, _b;
+  return typeof ((_b = (_a = manifest.packages) == null ? void 0 : _a[""]) == null ? void 0 : _b.version) === "string";
+}
 function isOptionalString(value) {
   const type = typeof value;
   return value === null || type === "undefined" || type === "string";
@@ -1190,6 +1194,9 @@ async function updateManifestFile(relPath, operation) {
   const file = await readJsonFile(relPath, cwd);
   if (isManifest(file.data) && file.data.version !== newVersion) {
     file.data.version = newVersion;
+    if (isPackageLockManifest(file.data)) {
+      file.data.packages[""].version = newVersion;
+    }
     await writeJsonFile(file);
     modified = true;
   }
diff --git a/node_modules/bumpp/dist/cli/index.js b/node_modules/bumpp/dist/cli/index.js
index 3c6d276..3e8d9c9 100644
--- a/node_modules/bumpp/dist/cli/index.js
+++ b/node_modules/bumpp/dist/cli/index.js
@@ -907,6 +907,10 @@ function writeTextFile(file) {
 function isManifest(obj) {
   return obj && typeof obj === "object" && isOptionalString(obj.name) && isOptionalString(obj.version) && isOptionalString(obj.description);
 }
+function isPackageLockManifest(manifest) {
+  var _a, _b;
+  return typeof ((_b = (_a = manifest.packages) == null ? void 0 : _a[""]) == null ? void 0 : _b.version) === "string";
+}
 function isOptionalString(value) {
   const type = typeof value;
   return value === null || type === "undefined" || type === "string";
@@ -1183,6 +1187,9 @@ async function updateManifestFile(relPath, operation) {
   const file = await readJsonFile(relPath, cwd);
   if (isManifest(file.data) && file.data.version !== newVersion) {
     file.data.version = newVersion;
+    if (isPackageLockManifest(file.data)) {
+      file.data.packages[""].version = newVersion;
+    }
     await writeJsonFile(file);
     modified = true;
   }
diff --git a/node_modules/bumpp/dist/cli/index.mjs b/node_modules/bumpp/dist/cli/index.mjs
index fd57a22..d26539b 100644
--- a/node_modules/bumpp/dist/cli/index.mjs
+++ b/node_modules/bumpp/dist/cli/index.mjs
@@ -7,7 +7,7 @@ import {
   log_symbols_default,
   require_picocolors,
   versionBump
-} from "../chunk-ZY4OVS4D.mjs";
+} from "../chunk-YUHUYJBY.mjs";
 
 // src/cli/index.ts
 import process2 from "process";
diff --git a/node_modules/bumpp/dist/index.js b/node_modules/bumpp/dist/index.js
index 785dedd..f8000a0 100644
--- a/node_modules/bumpp/dist/index.js
+++ b/node_modules/bumpp/dist/index.js
@@ -912,6 +912,10 @@ function writeTextFile(file) {
 function isManifest(obj) {
   return obj && typeof obj === "object" && isOptionalString(obj.name) && isOptionalString(obj.version) && isOptionalString(obj.description);
 }
+function isPackageLockManifest(manifest) {
+  var _a, _b;
+  return typeof ((_b = (_a = manifest.packages) == null ? void 0 : _a[""]) == null ? void 0 : _b.version) === "string";
+}
 function isOptionalString(value) {
   const type = typeof value;
   return value === null || type === "undefined" || type === "string";
@@ -1207,6 +1211,9 @@ async function updateManifestFile(relPath, operation) {
   const file = await readJsonFile(relPath, cwd);
   if (isManifest(file.data) && file.data.version !== newVersion) {
     file.data.version = newVersion;
+    if (isPackageLockManifest(file.data)) {
+      file.data.packages[""].version = newVersion;
+    }
     await writeJsonFile(file);
     modified = true;
   }
diff --git a/node_modules/bumpp/dist/index.mjs b/node_modules/bumpp/dist/index.mjs
index 366a4ef..ac60df9 100644
--- a/node_modules/bumpp/dist/index.mjs
+++ b/node_modules/bumpp/dist/index.mjs
@@ -7,7 +7,7 @@ import {
   loadBumpConfig,
   versionBump,
   versionBumpInfo
-} from "./chunk-ZY4OVS4D.mjs";
+} from "./chunk-YUHUYJBY.mjs";
 
 // src/index.ts
 var src_default = versionBump;
